bundles/binding/org.openhab.binding.powermax/src/main/java/org/openhab/binding/powermax/internal/connector/PowerMaxSerialConnector.java

```diff
@@ -0,0 +1,164 @@
+/**
+ * Copyright (c) 2010-2016, openHAB.org and others.
+ *
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+package org.openhab.binding.powermax.internal.connector;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.UnsupportedEncodingException;
+import java.util.TooManyListenersException;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import gnu.io.CommPort;
+import gnu.io.CommPortIdentifier;
+import gnu.io.NoSuchPortException;
+import gnu.io.PortInUseException;
+import gnu.io.SerialPort;
+import gnu.io.SerialPortEvent;
+import gnu.io.SerialPortEventListener;
+import gnu.io.UnsupportedCommOperationException;
+
+/**
+ * A class for the communication with the Visonic alarm panel through a serial connection
+ *
+ * @author lolodomo
+ * @since 1.9.0
+ */
+
+public class PowerMaxSerialConnector extends PowerMaxConnector {
+
+    private static final Logger logger = LoggerFactory.getLogger(PowerMaxSerialConnector.class);
+
+    private String serialPortName = "";
+    private int baudRate;
+
+    private SerialPort serialPort = null;
+
+    /**
+     * Constructor
+     *
+     * @param serialPortName
+     *            the serial port name
+     * @param baudRate
+     *            the baud rate to be used
+     */
+    public PowerMaxSerialConnector(String serialPortName, int baudRate) {
+        this.serialPortName = serialPortName;
+        this.baudRate = baudRate;
+    }
+
+    /**
+     * {@inheritDoc}
+     **/
+    @Override
+    public void open() {
+        logger.debug("open(): Opening Serial Connection");
+
+        try {
+
+            CommPortIdentifier portIdentifier = CommPortIdentifier.getPortIdentifier(serialPortName);
+            CommPort commPort = portIdentifier.open(this.getClass().getName(), 2000);
+
+            serialPort = (SerialPort) commPort;
+            serialPort.setSerialPortParams(baudRate, SerialPort.DATABITS_8, SerialPort.STOPBITS_1,
+                    SerialPort.PARITY_NONE);
+            serialPort.enableReceiveThreshold(1);
+            serialPort.disableReceiveTimeout();
+
+            setInput(serialPort.getInputStream());
+            setOutput(serialPort.getOutputStream());
+
+            getOutput().flush();
+            if (getInput().markSupported()) {
+                getInput().reset();
+            }
+
+            setReaderThread(new SerialReaderThread(getInput(), this));
+            getReaderThread().start();
+
+            setConnected(true);
+        } catch (NoSuchPortException noSuchPortException) {
+            logger.debug("open(): No Such Port Exception: {}", noSuchPortException.getMessage());
```

Comment:
Log as INFO level is done in PowerMaxBinding.openConnection to inform the user if connection succeeded or failed.